docs/architecture.md

```diff
@@ -1,27 +1,27 @@
 # Technical Architecture
 
-Netlify CMS is a React Application, using Redux for state management with immutable data structures (immutable.js).
+Netlify CMS is a React application, using Redux for state management with immutable data structures (immutable.js).
 
 The core abstractions for content editing are `collections`, `entries` and `widgets`.
 
-Each `collection` represents a collection of entries. This can either be a collection of similar entries with the same structure, or a set of entries where each has their own structure.
+Each `collection` represents a collection of entries. This can either be a collection of similar entries with the same structure, or a set of entries where each has its own structure.
 
 The structure of an entry is defined as a series of fields, each with a `name`, a `label` and a `widget` .
 
 The `widget` determines the UI widget that the content editor will use when editing this field of an entry, as well as how the content of the field is presented in the editing preview.
 
 Entries are loaded and persisted through a `backend` that will typically represent a `git` repository. 
 
-## State shape / reducers
+## State shape / reducers:
```

Comment:
Either is fine. Since its a header, leaving them off is a bit more standard. I'll go through and remove them if that's your preference.